README.md

```diff
@@ -13,16 +13,15 @@ that you can create `class`es and build methods that can perform
 actions on instance data, or specific to the `class`. What if you have
 `class`es that exhibit many of the same behaviors, such as `Cat`, `Dog`,
 and `Bird`, which all have a method for `speak`? In JavaScript, you can
-create "child" object `class`es (constructors) that inherit features
-from their "parent" `class`es. In this lesson, we'll discuss 2 ways of
-_extending_ functionality to other `class`es.
+create "child" object `class`es that inherit features from their "parent"
+`class`es. In this lesson, we'll discuss 2 ways of _extending_ functionality
```

Comment:
Better. I think there might be room for one more sentence after 'that inherit features from their "parent" `class`es.'  just to clarify what features. Or maybe it can just say 'that inherit _methods_ from their...'